contentcuration/contentcuration/viewsets/channel.py

```diff
@@ -119,28 +231,28 @@ class ChannelViewSet(ValuesViewset):
         "main_tree__created",
         "last_published",
         "ricecooker_version",
+        "main_tree__id",
     )
 
     field_map = {
         "thumbnail_url": get_channel_thumbnail,
         "published": "main_tree__published",
         "created": "main_tree__created",
+        "root_id": "main_tree__id"
     }
 
     def get_queryset(self):
+        user_id = not self.request.user.is_anonymous() and self.request.user.id
         queryset = Channel.objects.filter(deleted=False).filter(
-            id__in=Channel.objects.filter(
-                Q(editors=self.request.user)
-                | Q(viewers=self.request.user)
-                | Q(public=True)
-            )
-            .distinct()
-            .values_list("id", flat=True)
-        )
-        user_queryset = User.objects.filter(id=self.request.user.id)
+            Q(editors=user_id)
```

Comment:
```suggestion
            id__in=Channel.objects.filter(
                Q(editors=self.request.user)
                | Q(viewers=self.request.user)
                | Q(public=True)
            )
            .distinct()
            .values_list("id", flat=True)
        )
```

Reply:
I would suggest reverting this change for performance reasons.

Reply:
Odd, it won't let me commit this reversion - let's merge and follow this up.

Reply:
Ended up using `user_id` as one of the tests was failing (AnonymousUser not accepted for this query)